internal/hash/handler.go

```diff
@@ -0,0 +1,24 @@
+package hash
+
+import (
+	"context"
+	"github.com/mach-composer/mach-composer-cli/internal/config"
+	"github.com/mach-composer/mach-composer-cli/internal/graph"
+	"os"
+)
+
+const defaultHashFile = ".mach/hashes.json"
+
+type Handler interface {
+	Store(ctx context.Context, n graph.Node) error
+	Fetch(ctx context.Context, n graph.Node) (string, error)
+}
+
+func Factory(_ *config.MachConfig) Handler {
+	hashFile := os.Getenv("HASH_FILE")
```

Comment:
would it make sense to prefix the ENV?, e.g. MC_HASH_FILE ?

Reply:
Ah makes sense yes. I guess I should look into centralizing these and the `MCC_` variables with viper to make it a bit more clear what is coming from where. Will change this here though!